store-conf/conf/log4j.properties

```diff
@@ -1,142 +1,240 @@
-# An example log4j configuration file that outputs to System.out.  The
-# output information consists of relative time, log level, thread
-# name, logger name, nested diagnostic context and the message in that
-# order.
+status = error
+dest = err
+name = PropertiesConfig
+packages = com.zimbra
 
-# For the general syntax of property based configuration files see the
-# documenation of org.apache.log4j.PropertyConfigurator.
-
-log4j.threshhold=OFF
-
-log4j.rootLogger=INFO,CONSOLE,LOGFILE
-
-# CONSOLE is set to be a ConsoleAppender which outputs to System.out.
-log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
-log4j.appender.CONSOLE.layout=com.zimbra.common.util.ZimbraPatternLayout
-
-# The conversion pattern uses format specifiers. You might want to
-# change the pattern an watch the output format change.
-log4j.appender.CONSOLE.layout.ConversionPattern=%-4r %-5p [%t] [%z] %c{1} - %m%n
-
-# Appender LOGFILE writes to the file "mailbox.log".
+# Appender LOGFILE writes to the file "/opt/zimbra/log/mailbox.log".
```

Comment:
isn't this file working as global configuration for all logging? and not attached specifically with mailbox.log?

Reply:
This is a template file and we are not using it, just updated file with new sytax.

Reply:
> isn't this file working as global configuration for all logging? and not attached specifically with mailbox.log?

@silentsakky for your notes: https://github.com/Zimbra/zm-mailbox/blob/559986bc834ff8f6aca21eec87d2f8a4e2e4cf25/pkg-builder.pl#L241